qa-pipelines/tasks/lib/azure-aks.sh

```diff
@@ -46,22 +46,41 @@ azure_dns_clear() {
             --name ${matching_record_set}
     done
 }
+lbs_info() {
+    kubectl get svc -n $namespace -o json |
+    jq -c '
+    [
+      .items[] | select(.spec.type == "LoadBalancer") | {
+        "svc": .metadata.name,
+        "ip": .status.loadBalancer.ingress[0].ip,
+        "hostname": .status.loadBalancer.ingress[0].hostname
+       }
+    ]'
+}
+uaa_lb_info() {
+    kubectl get svc -n $namespace -o json |
+    jq -c '
+    [
+      .items[] | select(.spec.type == "LoadBalancer") |
+      select(.metadata.name == "uaa-uaa-public") | {
```

Comment:
Shouldn't we just always pass in the LB name, instead of having this weird mostly-duplicated function?
Or at least, call `lbs_info` and filter the results?